collections.js

```diff
@@ -22,25 +22,25 @@ function LinkedList() {
 
 LinkedList.prototype.empty = function() {
   return this.length == 0;
-}
+};
```

Comment:
These are not function declarations, rather they are regular assignment statements (that happen to use function literals) and all assignment statements should end in a semicolon.